skimage/morphology/extrema.py

```diff
@@ -415,28 +414,28 @@ def local_maxima(image, selem=None, connectivity=None, indices=False,
     connectivity):
 
     >>> local_maxima(image)
-    array([[0, 0, 0, 0, 0, 0, 0],
-           [0, 1, 1, 0, 0, 0, 0],
-           [0, 1, 1, 0, 0, 0, 0],
-           [1, 0, 0, 0, 0, 0, 1]], dtype=uint8)
+    array([[False, False, False, False, False, False, False],
+           [False,  True,  True, False, False, False, False],
+           [False,  True,  True, False, False, False, False],
+           [ True, False, False, False, False, False,  True]], dtype=bool)
```

Comment:
Typically in these situations I would use `local_maxima(image).astype(int)` so that the printout is readable. But this is (a) not critical, and (b) debatable anyway. Just thought I would put it out there.

Reply:
Yeah, I'm on the fence as well. `astype(int)` would make the printed output more readable but hide the fact that the returned array is of boolean type. I'm +0 on leaving it like this.

Reply:
I'd be fine with it as-is.  Another option would be changing the print options at the top of the docstring example, like

``` python
>>> np.set_printoptions(formatter={'bool':lambda x: str(int(x))})
```

so no need to do `.astype()` or `.view()` on every line.